ci/envs/311-latest-conda-forge.yaml

```diff
@@ -0,0 +1,36 @@
+name: test
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  # required
+  - pandas
+  - shapely
+  - fiona
+  - tiledb=2.11.3
```

Comment:
I suppose this isn't actually needed anymore

Reply:
Yeah, doesn't seem so. Removed from all envs.